src/relentless/simulate/hoomd.py

```diff
@@ -1533,7 +1538,7 @@ class HOOMD(simulate.Simulation):
     will be automatically selected for you when the simulation is run. Both
     HOOMD 2.x and 3.x are supported.
 
-    The `freud <https://freud.readthedocs.io>`_ analysis package (version 2.x)
```

Comment:
`freud` is actually a dependency for all of relentless, so let's just remove this discussion here. Leave the parts about hoomd.